src/routes/forms/submission/router.ts

```diff
@@ -10,4 +10,7 @@ export const submissionApiRoute = Router({
 submissionApiRoute
   .use("/new", newApiRoute)
   .use("/downloaded", downloadedApiRoute)
-  .use("/:submissionId", submissionIdApiRoute);
+  .use(
+    "/:submissionName([0-9]{2}-[0-9]{2}-[a-z0-9]{4})",
```

Comment:
I saw the FormID validation—sure, why not, it prevents passing in a garbage ID. But now I'm wondering if we're locking ourselves in, especially since our system footprint is growing.

Reply:
I mean this is the format of a submission name as designed in our system so I would not see any reason why not to have this validation. If it changes in the future then we can revisit (could also be something that changes in a v2).
The good thing with this check is that he prevents malicious users from sending DynamoDB requests that we already know will fail.